src/util/seed.py

```diff
@@ -1,6 +1,5 @@
 import random
 import numpy as np
-import scipy
```

Comment:
Why did the soldiers out it in here for the first place?

Reply:
probably a mistake. scipy.random doesn't even exist